src/script/backport-resolve-issue

```diff
@@ -487,7 +487,7 @@ Ceph version:     base {}, target {}'''.format(self.github_url, pr_title_trunc,
 
     def extract_backport_trackers_from_github_pr_desc(self):
         global redmine_endpoint
-        p = re.compile('http.*://tracker.ceph.com/issues/\\d+')
+        p = re.compile('http.://tracker.ceph.com/issues/\\d+')
```

Comment:
This will force the url to start with `https`, since the `.` is not optional.
By adding a `?` it will also catch `http`:
```suggestion
        p = re.compile('http.?://tracker.ceph.com/issues/\\d+')
```

Reply:
Thanks for this!